k8s.gcr.io/images/k8s-staging-kustomize/OWNERS

```diff
@@ -4,12 +4,13 @@
 # https://github.com/kubernetes/org/blob/master/config/kubernetes-sigs/sig-cli/teams.yaml.
```

Comment:
This comments states that the list is supposed to be mirroring https://github.com/kubernetes/org/blob/main/config/kubernetes-sigs/sig-cli/teams.yaml#L91-L106, which @monopole recently updated. Can you make it do that, or if that's not a good idea for some reason, change the comment?

Reply:
I've removed the comment. I don't see a reason not to update it to mirror the other list but I'm not sure how to do that.

> Are you in kubernetes-sigs but not kubernetes, and if that's the case, any reason not to get that fixed?

This is probably the case. Is adding me to the `kubernetes` org something you are able to do?

Reply:
> I don't see a reason not to update it to mirror the other list but I'm not sure how to do that.

I don't mean with automation, just copy-paste 😄 

> This is probably the case. Is adding me to the kubernetes org something you are able to do?

The process is to [open an issue](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E) on the org repo. I can sponsor you, and I expect @monopole would be willing to too

Reply:
I am happy to sponsor as well @natasha41575

Reply:
@dims @KnVerey thanks! opened the issue here: https://github.com/kubernetes/org/issues/2789